### PR TITLE
docs/contributing: add info for replicated's about gcp/codeserver env

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

Allow those that are new member easily get the link/info for who to setup the env described to do the tests

#### Which issue(s) this PR fixes:
NONE

#### Special notes for your reviewer:
NONE

## Steps to reproduce
NONE

#### Does this PR introduce a user-facing change?
NONE

#### Does this PR require documentation?
NONE

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,7 +106,9 @@ Testing can be accomplished on systems capable of hosting supported container ru
 
 ### Using GCP
 
-Convenience functions (change the Default name, the first variable):
+**If you are a Replicated team member then:** ensure that you follow up the doc [Getting Started with your own CodeServer][code-server].
```
Comment (laverya) [inline CONTRIBUTING.md:109]:
```suggestion
**If you are a Replicated team member then:** ensure that you have followed [Getting Started with your own CodeServer](https://github.com/replicatedhq/codeserver/blob/main/docs/first-time.md).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -202,3 +204,4 @@ Snapshot management requires VM t be powered off.
 
 ### QEMU on MacOS
 
+[code-server]: https://github.com/replicatedhq/codeserver/blob/main/docs/first-time.md
```
Comment (laverya) [inline CONTRIBUTING.md:207]:
```suggestion
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (camilamacedo86) [reply CONTRIBUTING.md:109]: Use alias make easier we keep the docs maintained since all links would be at the bottom

Comment (camilamacedo86) [reply CONTRIBUTING.md:109]: However, it is not a big deal eaither.

**Review by laverya — APPROVED** [review]

**Pull request merged**